sycl/doc/extensions/experimental/sycl_ext_oneapi_graph.asciidoc

```diff
@@ -245,15 +271,60 @@ represents either a command-group or an empty operation.
 through newly added interfaces. This is either using the `make_edge()` function
 to define an edge between existing nodes, or using a
 `property::node::depends_on` property list when adding a new node to the graph.
+Nodes or <<event-terminology, graph-limited events>> passed to this property
+that are from the same graph will create edges within the graph.
 
 Edges can also be created when explicitly adding nodes to the graph through
 existing SYCL mechanisms for expressing dependencies. Data dependencies from
-accessors to existing nodes in the graph are captured as an edge. Using
-`handler::depends_on()` will also create a graph edge when passed an event
-returned from a queue submission captured by a queue recording to the same
-graph.
+accessors to existing nodes in the graph are captured as an edge. 
+
+Using `handler::depends_on()` inside the node's command-group function can also
+be used for defining graph edges. However, for an event passed to
+`handler::depends_on()` to create an edge, it must be an event returned from a
+queue submission captured by the same graph (a <<event-terminology,
+graph-limited event>>). 
+
+| External Dependencies | Graph nodes may have dependencies on operations
+outside of the graph they belong to. These can be dependencies on nodes from
+<<inter-graph-dependencies, another graph>> or on eager SYCL operations. Passing
+<<event-terminology, graph-limited events>> from another graph or
+<<event-terminology, regular SYCL events>> to `handler::depends_on()`, as well
+as passing nodes from another graph to the `property::node::depends_on`
```

Comment:
The case of passing <<event-terminology, graph-limited events>> from another graph or <<event-terminology, regular SYCL events>> to `property::node::depends_on` is not covered.

Reply:
I've clarified the wording to clarify that these can also be passed to the property.